packages/common/package.json

```diff
@@ -45,7 +45,7 @@
     "chalk-pipe": "^3.0.0",
     "dotenv": "^6.2.0",
     "eth-crypto": "^1.7.0",
-    "hardhat": "^2.4.0",
+    "hardhat": "^2.4.3",
     "hardhat-deploy": "^0.8.6",
```

Comment:
Can you also bump `hardhat-deploy` to 0.8.11 which includes mumbai + polygonscan compatibility?

Reply:
I've updated the package and yarn accordingly.